contracts/MemeArbitrum.sol

```diff
@@ -9,14 +9,22 @@ interface IWETH {
 
 // @title MemeArbitrum - a smart contract factory for Meme Token creation on Arbitrum.
 contract MemeArbitrum is MemeFactory {
+    // Base UniswapV3 pool cardinality that corresponds to 240 seconds window (240 / 1 seconds per block)
+    uint16 public constant POOL_CARDINALITY = 240;
```

Comment:
This affects gas usage a lot. Have communicated with auditors for these new numbers across all the networks:
240 for Arbitrum
120 for Base
60 for Celo and ETH